oojs.md

```diff
@@ -7,15 +7,25 @@ Define a `Playlist` class. It should have...
 - An `addSong` (Function) method that adds a song to the `songs` property
 
 ```js
-Your answer goes here
+class Playlist={
+  constructor(title="",songs=[]){
+    this.title=title;
+    this.songs=songs
+  }   
+  favorites:0;
+   addSong:function(){
+     Playlist.push(songs)
```

Comment:
- `Playlist` is a class, not an array. You need to push into a particular instance's `songs` array
- This method needs a parameter to represent the song that is being added

```js
addSong: function(newSong){
  this.songs.push(newSong)
}
```